Src/coffeescript/cql-execution/src/runtime/context.coffee

```diff
@@ -84,6 +84,33 @@ module.exports.Context = class Context
   set: (identifier, value) ->
     @context_values[identifier] = value
 
+  setLocalIdWithResult: (localId, value) ->
+    @localId_context[localId] = value
+
+  getLocalIdResult: (localId) ->
+    @localId_context[localId]
+
+  # Returns an object of objects containing each library name
+  # with the localIds and result values
+  getAllLocalIds: ->
+    localIdResults = {}
+    # Add the localIds and result values from the main library
+    localIdResults[@parent.source.library.identifier.id] = {}
+    for localId, value of @localId_context
+      localIdResults[@parent.source.library.identifier.id][localId] = value
```

Comment:
Here you are iterating `@localId_context` and storing the key/value in `localIdResults[@parent.source.library.identifier.id]` one by one.

In `supportLibraryLocalIds`, you are are just assigning `lib.localId_context` to `localIdResults[lib.library.source.library.identifier.id]`.

In both cases you are essentially trying to do the same thing -- is there a reason for the difference?